anki/tags.py

```diff
@@ -102,7 +102,7 @@ def bulkAdd(self, ids, tags, add=True):
         res = self.col.db.all(
             "select id, tags from notes where id in %s and (%s)" % (
                 ids2str(ids), lim),
-            **dict([("_%d" % x, '%% %s %%' % y)
+            **dict([("_%d" % x, '%% %s %%' % y.replace('*', '%'))
                     for x, y in enumerate(newTags)]))
```

Comment:
Perhaps worth replacing this by a dict comprehension (and a more modern .format with keywords), while we're there? The existing line isn't very readable...